_posts/2022-01-14-quorn-a-story-about-single-cell-protein.md

```diff
@@ -0,0 +1,197 @@
+
+At a high level, we ended up with a bioprocess that is more efficient than animal rearing, cost and nutritionally competitive as a meat substitute, but still more expensive than plant proteins. If we examine the caloric and protein conversion efficiencies of Quorn next to animal proteins [13], we can see that it holds up well enough:
+
+
+{:.table-clean}
+| Product           | Caloric Conversion Efficiency | Protein Conversion Efficiency |
+| ----------------- | ----------------------------- | ----------------------------- |
+| Quorn             | 21%                           | N/D                           |
+| Eggs (Chicken)    | 17%                           | 31%                           |
+| Milk (Dairy Cows) | 17%                           | 14%                           |
+| Poultry           | 13%                           | 21%                           |
+| Pork              | 9%                            | 9%                            |
+| Beef              | 3%                            | 3%                            |
+
+The efficiency of Quorn is hampered by the 30% biomass loss due to RNA reduction - in this table I've only considered the centrifuged cake as the consumable end-product. This might be the Achilles' Heel of all SCP - you can have a microorganism that either **grows slowly**, or has **high nucleic acids** - not both. And removing nucleic acids comes at a hefty cost. 
+
+Quorn doesn't really have a protein conversion efficiency because it directly utilizes ammonia, not protein, as a nitrogen source. 
+
+##### Quorn as a Pathway
+
+Having a direct pathway from **carbohydrates + ammonia** to **protein** has a lot of neat implications. If plant-based meat substitutes continue to gain traction (and I think they will), the production of plant protein isolates and concentrates is going to yield a lot of waste carbohydrates and starches, which will become dirt cheap. This would make the Quorn process a lucrative pathway to upgrade cheap carbohydrates into protein.
+
+Alternatively, if corn ethanol gets banned due to policy changes, we're going to have a lot of un-used corn milling and hydrolysing infrastructure. These could easily be adapted into Quorn-type protein production plants. 
+
+**Taste and Texture Matter**
+
+A food products' **organoleptic properties** matter a lot in the meat substitute space. Quorn's success comes in part due to the fact the the hyphal product, after heat treatment and freezing, resembles the fibrous texture of meat. 
+
+![](/assets/images/quorn/quorn-meatless-pieces-fried-texture.png){:height="350" .center}
```

Comment:
I added an image here - the paragraph encourages the imagination of what quorn looks like, might as well how the audience!

Reply:
Good call. If only I could buy some and take my own picture :(